.github/workflows/cd-binaries.yaml

```diff
@@ -42,53 +43,12 @@ jobs:
         with:
           distribution: goreleaser
           version: latest
-          args: release --rm-dist --skip-publish --snapshot
+          args: release --rm-dist --snapshot ${{ startsWith(github.ref, 'refs/tags/') || '--skip-publish' }}
```

Comment:
This allows CI to still build the binaries and packages without publishing them. Useful for running in `main`

Reply:
Great. Does this mean we'll have binary artifacts for every commit on `main` (great for testing)

Reply:
We had this before as well. Those artifacts only live in GitHub actions though